include/deal.II/lac/precondition.h

```diff
@@ -301,13 +301,18 @@ class PreconditionUseMatrix : public Subscriptor
  * Jacobi, SOR and SSOR preconditioners are implemented. For preconditioning,
  * refer to derived classes.
  *
- * @author Guido Kanschat, 2000
+ * @author Guido Kanschat, 2000 ; Jean-Paul Pelteret 2015
  */
 template<class MATRIX = SparseMatrix<double> >
 class PreconditionRelaxation : public Subscriptor
 {
 public:
   /**
+   * Declare type for container size.
+   */
+  typedef types::global_dof_index size_type;
```

Comment:
I think the right choice would be to import the size_type of the matrix template argument of this class.